bigtable/bttest/inmem.go

```diff
@@ -225,6 +278,9 @@ func (s *server) ModifyColumnFamilies(ctx context.Context, req *btapb.ModifyColu
 			tbl.counter++
 			tbl.families[mod.Id] = newcf
 		} else if mod.GetDrop() {
+			if tbl.isProtected {
+				return nil, status.Errorf(codes.FailedPrecondition, "table %q is protected from deletion", req.Name)
```

Comment:
This is not consistent with ours server implementation I believe.

It should be like this:

if tbl.isProtected && request has at least one drop cf
    return FailedPrecondition

// Then start updating column families
for _, mod := range req.Modifications {

Reply:
Ack. My reading of the protos implies that the two statements are (functionally) equivalent.

The `else if mod.GetDrop()` will evaluate as true if the request has at least one drop. The `if tlb.isProtected`, branch of the next statement thus returns an error if BOTH the conditions are true.